src/app/api/tickets/route.ts

```diff
@@ -0,0 +1,23 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+
+import prisma from '@/common/lib/prisma';
+
+export const POST = async (req: NextApiRequest, res: NextApiResponse) => {
+    const { userId } = await req.body;
+
+    try {
+        const userTickets = await prisma.totoTicket.findMany({
+            where: {
+                uuid: userId
```

Comment:
there's red squiggly on this `uuid`. changing it to `id` seems to fix it. is there a reason for using `uuid`? or is there a way to make `uuid` the default?

Reply:
There isnt a red squiggly on my side, but the uuid is the field defined in the prisma schema. This issue could be because the prisma schema was updated in the schema, but not in you local env. 
 
Maybe running 'npx prisma generate' on your end will rectify this issue